srcpkgs/nqp/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'nqp'
 pkgname=nqp
-version=2018.09
+version=2018.10
 revision=1
 build_style=configure
 make_check_target=test
 configure_script="perl Configure.pl"
 configure_args="--prefix=/usr --backends=moar"
 hostmakedepends="perl"
-makedepends="MoarVM"
-depends="MoarVM"
+makedepends="MoarVM=2018.10"
+depends="MoarVM=2018.10"
```

Comment:
correct specifier for exact version is `-`

other available are `>=` `<=` `>` `<`

Reply:
`-` was what I went with originally: https://travis-ci.org/void-linux/void-packages/jobs/447580109

I assume I needed to add `_revision`, then